src/components/gameScreen/gameScreen.container.js

```diff
@@ -115,41 +94,39 @@ export const GameScreenContainer = () => {
 
   const bookmarkSetter = async () => {
     try {
-      if (await AsyncStorageService.setBookmark({
-      question: currentQuestion,
-      help: filteredQuestion[0].help,
-      rightAnswer: filteredQuestion[0].rightAnswer,
-    })) setIsBookmarkSet(true);
-      else {setIsBookmarkSet(false)}
+      const isBookmarkSet = await AsyncStorageService.setBookmark({
+        question: store.currentQuestion.text,
+        help: store.currentQuestion.help,
+        rightAnswer: store.currentQuestion.rightAnswer,
+      })
+
+      isBookmarkSet ?
+        bookmarkStore.setIsBookmarkSet(true)
+        :
+        bookmarkStore.setIsBookmarkSet(false)
```

Comment:
ternary operators are used only for assignment and not for conditional function invokation.
This can be written like: 
`bookmarkStore.setIsBookmarkSet(isBookmarkSet)`